contract-tests/tests/test/amazon/botocore/botocore_test.py

```diff
@@ -176,8 +176,6 @@ def test_dynamodb_create_table(self):
             0,
             remote_service="AWS::DynamoDB",
             remote_operation="CreateTable",
-            remote_resource_type="AWS::DynamoDB::Table",
```

Comment:
I don't think this is expected behaviour. AWS SDK remote resource info should not be impacted.